app/models/changeset/commit.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 class Changeset::Commit
-  PULL_REQUEST_MERGE_MESSAGE = /\AMerge pull request #(\d+)/.freeze
-  PULL_REQUEST_SQUASH_MESSAGE = /\A.*\(#(\d+)\)$/.freeze
+  PULL_REQUEST_MESSAGE = /\A.*\(#(\d+)\)$/.freeze
```

Comment:
no longer matches `pull request #123`

Reply:
True 😊
Should be changed 👍

Reply:
Fixed 😉

Thanks a lot for your vigilance 🙏